common/common.h

```diff
@@ -32,22 +32,32 @@ template <class E> using UnorderedSet = absl::flat_hash_set<E>;
 
 #define _MAYBE_ADD_COMMA(...) , ##__VA_ARGS__
 
+// A faster version of ENFORCE that does not emit a timer. Useful for checks that happen extremely frequently and
+// are O(1). Please avoid using unless ENFORCE shows up in profiles.
+#define ENFORCE_FAST(x, ...)                                                                                \
```

Comment:
We encourage people to prefer the fast thing many times in Sorbet (e.g., people cargo cult `fast_sort`).

I worry that people will think `ENFORCE_FAST` should be their default choice if they don't read the comment. What do you think about `UNTIMED_ENFORCE` or `ENFORCE_WITHOUT_TIMER`?

Reply:
ah just saw your comment (https://github.com/sorbet/sorbet/pull/3341#issuecomment-668310407). That name works for me too!